sidebars.js

```diff
@@ -3,26 +3,16 @@ module.exports = {
     'Getting Started': [
       'intro',
       'installation',
+      'building',
     ],
     'Mapping Elements': [
-      'maps/mapml-viewer',
-      'maps/web-map',
-      {
-        'Layers': [
-          'layers/layer',
-          'layers/static-tiles',
-          'layers/static-features',
-          'layers/static-images',
-          'layers/templated-tiles',
-          'layers/templated-features',
-          'layers/templated-images',
-        ],
```

Comment:
Looks good so far! I was wondering how you plan on presenting the different layer types as I see they are removed from the menu in the changes.

Reply:
I was thinking of example pages or tutorial pages.

Reply:
Ahh okay, that's a good way to organize it.